docs/docs/behaviors/lighting.md

```diff
@@ -3,12 +3,27 @@ title: Lighting Behavior
 sidebar_label: Lighting
 ---
 
+import Tabs from '@theme/Tabs';
```

Comment:
I'm not sure it's a good idea to use tabs here. If I give you a link to something inside the second tab like https://deploy-preview-904--zmk.netlify.app/docs/behaviors/lighting/#behavior-binding-1, it won't automatically switch to the correct tab. The table of contents on the right is also confusing because it doesn't see the tabs as a header level.